File: scratch/pup/fix-div.py
import re
with open('app/global-reach/page.tsx', 'r', encoding='utf-8') as f:
    code = f.read()

# Replace 4 divs with 3 divs
pattern = r'spanning continents\.\s*</p>\s*</div>\s*</div>\s*</div>\s*</div>\s*</Section>'
replacement = '''spanning continents.
                </p>
              </div>
            </div>
          </div>
        </Section>'''

new_code, count = re.subn(pattern, replacement, code)
if count > 0:
    with open('app/global-reach/page.tsx', 'w', encoding='utf-8') as f:
        f.write(new_code)
    print(f'Fixed! Replacements: {count}')
else:
    print('Not found!')

File: scratch/pup/fix-div.js
const fs = require('fs');
let code = fs.readFileSync('app/global-reach/page.tsx', 'utf8');

const broken = `                {/* Mobile fallback */}
                <p className="text-white/80 pt-4 max-w-sm font-light text-sm md:text-base leading-relaxed lg:hidden">
                  Direct logistics and distribution networks spanning continents.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </Section>`;

const fixed = `                {/* Mobile fallback */}
                <p className="text-white/80 pt-4 max-w-sm font-light text-sm md:text-base leading-relaxed lg:hidden">
                  Direct logistics and distribution networks spanning continents.
                </p>
              </div>
            </div>
          </div>
        </Section>`;

code = code.replace(broken, fixed);
fs.writeFileSync('app/global-reach/page.tsx', code);
console.log('Fixed div count!');
